packages/visual-editor/src/components/Dropzone/EditorBlock.tsx

```diff
@@ -55,6 +55,7 @@ export const EditorBlock: React.FC<EditorBlockProps> = ({
   const sectionsWithZone = useZoneStore((state) => state.sectionsWithZones);
 
   const isContainer = node.data.blockId === CONTENTFUL_CONTAINER_ID;
+  const isSingleColumn = node.data.blockId === CONTENTFUL_SINGLE_COLUMN_ID;
```

Comment:
I just merged in a change that deprecated these constants.

```ts
import { CONTENTFUL_COMPONENTS } from '@contentful/experience-builder';
```

```suggestion
  const isContainer = node.data.blockId === CONTENTFUL_COMPONENTS.container.id;
  const isSingleColumn = node.data.blockId === CONTENTFUL_COMPONENTS.singleColumn.id;
```